app/assets/javascripts/users.js.erb

```diff
@@ -220,9 +225,37 @@ function deleteTargetEvents() {
   });
 }
 
-$(document).ready(function() {
-  initMap();
+function activateMapEvents() {
+  $('.welcome-screen').addClass('hidden');
   displayTargets();
   createTargetEvents();
   deleteTargetEvents();
+}
+
+function editFirstLoginEvent() {
+  $('.welcome-bottom .principal-button').click(function() {
+    var route = $('.welcome-button.principal-button').data('edit-user-path');
```

Comment:
The button which trigger the action is the same that you use for extract the file right? If it's the case you can do `var route = $(this).data('edit-user-path');`